core/src/main/java/org/mobilitydata/gtfsvalidator/validator/DefaultFieldValidator.java

```diff
@@ -93,6 +93,11 @@ public void validateEmail(
   @Override
   public void validatePhoneNumber(
       String phoneNumber, GtfsCellContext cellContext, NoticeContainer noticeContainer) {
+    if (countryCode.isUnknown()) {
+      // If the user didn't provide a country code or the provided country-code was invalid, skip
+      // validation
```

Comment:
Please put a full stop at the end of comments. They must be complete sentences.

Reply:
Done in https://github.com/MobilityData/gtfs-validator/pull/1062/commits/5223228e99241ccf2e3395c5822fe8abdf0104cd.